node/src/components/consensus/protocols/highway.rs

```diff
@@ -55,6 +55,8 @@ const TIMER_ID_PURGE_VERTICES: TimerId = TimerId(2);
 const TIMER_ID_LOG_PARTICIPATION: TimerId = TimerId(3);
 /// The timer for an alert no progress was made in a long time.
 const TIMER_ID_STANDSTILL_ALERT: TimerId = TimerId(4);
+/// The timer for logging synchronizer queue size.
+const TIMER_ID_SYNCHRONIZER_QUEUE: TimerId = TimerId(5);
```

Comment:
```suggestion
const TIMER_ID_SYNCHRONIZER_LOG: TimerId = TimerId(5);
```
Or `QUEUE_LOG` or something?